dev/conductor/core/lib/src/next.dart

```diff
@@ -149,7 +149,7 @@ class NextContext {
           );
           if (!response) {
             stdio.printError('Aborting command.');
-            writeStateToFile(stateFile, state, stdio.logs);
+            state_import.writeStateToFile(stateFile, state, stdio.logs);
```

Comment:
Should this use `updateState(stateFile, state, stdio.logs)` as well?

Reply:
good catch

Reply:
updated, PTAL